tests/automation_framework/src/libs/test_base.py

```diff
@@ -49,7 +47,7 @@ def setup_and_build_request_wo_getresult(self, input_file):
              'action_obj': wo_submit})
         return 0
 
-    def setup_and_build_request_retrieve(self, input_file):
+    def setup_and_build_request_worker_retrieve(self, input_file):
```

Comment:
Renamed the setup_and_build_request_retrieve to setup_and_build_request_worker_retrieve